experiments/run-benchmark.py

```diff
@@ -76,12 +76,17 @@ def compute_metrics(
 
     else:
         index.set_ef(ef_search)
-        for query in queries:
+        for idx, query in enumerate(queries):
             start = time.time()
             indices, _ = index.knn_query(data=np.array([query]), k=k)
             end = time.time()
             latencies.append(end - start)
             top_k_indices.append(indices[0])
+            
+            if idx == len(queries) - 1:
+                # HNSW aggregates distance computations across all queries.
+                distance_computations.append(index.get_distance_computations())
```

Comment:
This is the main change.

Reply:
Minor: Consider using `for-else` loop here https://www.w3schools.com/python/gloss_python_for_else.asp

Reply:
This looks much neat! didn't know for-else existed:)